endpoints-support.md

```diff
@@ -35,3 +35,4 @@ Refer SimpliVity REST API doc for the resource endpoints documentation [HPE Simp
 |<sub>/virtual_machines/{vmId}/clone	</sub>                                            |POST      |
 |<sub>/virtual_machines/{vmId}/move	</sub>                                                |POST      |
 |<sub>/virtual_machines/{vmId}/set_policy	</sub>                                        |POST      |
+|<sub>/virtual_machines/{vmId}/power_off	</sub>                                        |POST      |
```

Comment:
move up one to keep it in alphabetical order

Reply:
@svt-hchaudhari can you follow up on this request?

Reply:
Updated as per the alphabetical order